massa-node/src/main.rs

```diff
@@ -339,7 +341,13 @@ async fn launch(
         block_count_considered: MIP_STORE_STATS_BLOCK_CONSIDERED,
         counters_max: MIP_STORE_STATS_COUNTERS_MAX,
     };
-    let mip_store = MipStore::try_from(([], mip_stats_config)).unwrap();
+    let mut mip_store =
+        MipStore::try_from(([], mip_stats_config)).expect("Cannot create an empty MIP store");
```

Comment:
I'm not sure I remember the design choice here. When we want to add a MIP, the devs would just include it in the store init here?
And after that the update_with below will check compatibilities between our MIP store and the MIP store of our bootstrap server?

Reply:
Ok!